packages/odf/components/s3-browser/bucket-overview/BucketOverview.tsx

```diff
@@ -0,0 +1,213 @@
+import * as React from 'react';
+import { LoadingBox } from '@odf/shared/generic/status-box';
+import PageHeading from '@odf/shared/heading/page-heading';
+import { useRefresh } from '@odf/shared/hooks';
+import { ModalKeys, defaultModalMap } from '@odf/shared/modals/types';
+import { BlueSyncIcon } from '@odf/shared/status';
+import { K8sResourceKind } from '@odf/shared/types';
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import {
+  referenceForModel,
+  getValidWatchK8sResourceObj,
+} from '@odf/shared/utils';
+import { YAMLEditorWrapped } from '@odf/shared/utils/Tabs';
+import {
+  useK8sWatchResource,
+  HorizontalNav,
+  useModal,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { LaunchModal } from '@openshift-console/dynamic-plugin-sdk/lib/app/modal-support/ModalProvider';
+import { TFunction } from 'i18next';
+import { useNavigate, NavigateFunction } from 'react-router-dom-v5-compat';
+import { useParams, useSearchParams } from 'react-router-dom-v5-compat';
+import { Button, ButtonVariant } from '@patternfly/react-core';
+import { ActionsColumn, IAction } from '@patternfly/react-table';
+import { PREFIX, BUCKETS_BASE_ROUTE } from '../../../constants';
+import { NooBaaObjectBucketModel } from '../../../models';
+import { getBreadcrumbs } from '../../../utils';
+import { NoobaaS3Provider } from '../noobaa-context';
+import { CustomActionsToggle, ObjectsList } from '../objects-list';
+import { PageTitle } from './PageTitle';
+
+const getBucketActionsItems = (
+  t: TFunction,
+  launcher: LaunchModal,
+  navigate: NavigateFunction,
+  bucketName: string,
+  isCreatedByOBC: boolean,
+  noobaaObjectBucket: K8sResourceKind
+): IAction[] => [
+  // ToDo: add empty/delete bucket actions
+  {
+    title: t('Empty bucket'),
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    onClick: () => {},
```

Comment:
You don't need the disable comment if you return `undefined` (according to the TypeScript documentation, a void return type is equivalent to a return type of `undefined`):
```suggestion
    onClick: () => undefined,
```